README.md

```diff
@@ -46,7 +46,7 @@ The NAB scores are normalized such that the maximum possible is 100.0 (i.e. the
 
 \* The results correspond to NuPIC and nupic.core SHAs 42f701d and c030b84 respectively, but the latest version of NuPIC should still work (the results may not be identical).
 
-** Details of the implementation and parameter tuning are in the [detector's code](https://github.com/numenta/NAB/blob/master/nab/detectors/bayes_changept/bayes_changept_detector.py).
+** The original algorithm is not designed for anomaly detection, but can be used as such -- changepoints can be anomalies. Details of the implementation and parameter tuning are in the [detector's code](https://github.com/numenta/NAB/blob/master/nab/detectors/bayes_changept/bayes_changept_detector.py).
```

Comment:
Suggestion:  "The original algorithm was not designed for anomaly detection. Details of the anomaly detection implementation and parameter tuning are in the ..."